web_login_default_logo/controllers/main.py

```diff
@@ -21,8 +20,4 @@ class Binary(http.Controller):
         cors="*",
     )
     def no_logo(self, dbname=None, **kw):
-        placeholder = functools.partial(
-            get_resource_path, "web", "static", "src", "img"
-        )
-
-        return http.send_file(placeholder("nologo.png"))
+        return http.Stream(file_path("web/static/img/nologo.png"))
```

Comment:
It'd be better to us this `http.Stream.from_path(` 
Please check this example:
https://github.com/odoo/odoo/blob/a22c454655b0c6e01af9764c4552bf94469b9d02/addons/web/controllers/binary.py#L262